Sources/ArcGISToolkit/Components/Popups/PopupMedia/Charts/ChartMediaView.swift

```diff
@@ -94,25 +91,29 @@ struct ChartView: View {
     }
     
     var body: some View {
-        switch popupMedia.kind {
-        case .barChart, .columnChart:
-            BarChart(
-                chartData: data,
-                isColumnChart: (popupMedia.kind == .columnChart),
-                isShowingDetailView: isShowingDetailView
-            )
-        case .pieChart:
-            PieChart(
-                chartData: data,
-                isShowingDetailView: isShowingDetailView
-            )
-        case .lineChart:
-            LineChart(
-                chartData: data,
-                isShowingDetailView: isShowingDetailView
-            )
-        default:
-            EmptyView()
+        VStack {
+            switch popupMedia.kind {
+            case .barChart, .columnChart:
+                BarChart(
+                    chartData: data,
+                    isColumnChart: (popupMedia.kind == .columnChart),
+                    isShowingDetailView: isShowingDetailView
+                )
+            case .pieChart:
+                PieChart(
+                    chartData: data,
+                    isShowingDetailView: isShowingDetailView
+                )
+            case .lineChart:
+                LineChart(
+                    chartData: data,
+                    isShowingDetailView: isShowingDetailView
+                )
+            default:
+                EmptyView()
+            }
+            Spacer()
```

Comment:
What is the reasoning for this spacer? I'm not noticing a difference with/without it on an iOS simulator.